Source/UserManagement/Domain/DataCollectors/Validators/AddPhoneNumberToDataCollectorValidator.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using Domain.DataCollectors.Commands;
+using FluentValidation;
+
+namespace Domain.DataCollectors.Validators
+{
+    public class AddPhoneNumberToDataCollectorValidator : AbstractValidator<AddPhoneNumberToDataCollector>
+    {
+        public AddPhoneNumberToDataCollectorValidator()
+        {
+            CascadeMode = CascadeMode.StopOnFirstFailure;
```

Comment:
This is a global validation mode.  I prefer to get all the issues with the command so that these can be displayed to the user.  

Sometimes you have multiple rules on a single property that depends on it not being null for example.  Then you can set the Cascade mode on the property and have the null check first.